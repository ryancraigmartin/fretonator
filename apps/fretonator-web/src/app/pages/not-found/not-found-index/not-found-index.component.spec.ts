import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { Component } from '@angular/core';
import { NotFoundModule } from '../not-found.module';
import { RouterTestingModule } from '@angular/router/testing';

describe('NotFoundIndexComponent', () => {
  @Component({
    selector: 'app-not-found-index-spec',
    template: `
      <app-not-found-index></app-not-found-index>
    `
  })
  class NotFoundIndexSpecComponent {
  }

  let component: NotFoundIndexSpecComponent;
  let fixture: ComponentFixture<NotFoundIndexSpecComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [NotFoundIndexSpecComponent],
      imports: [
        RouterTestingModule,
        NotFoundModule
      ]
    })
      .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(NotFoundIndexSpecComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
    expect(fixture).toMatchSnapshot();
  });
});
