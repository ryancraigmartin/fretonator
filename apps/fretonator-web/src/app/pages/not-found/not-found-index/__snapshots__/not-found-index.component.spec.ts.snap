// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundIndexComponent should create 1`] = `
<app-not-found-index-spec
  __ngContext__={[Function LRootView]}
>
  <app-not-found-index>
    <main
      class="notFound__container"
    >
      <div
        class="notFound__inner"
      >
        <h1
          class="notFound__title"
        >
          Broken string 404!
        </h1>
        <h2
          class="notFound__subtitle"
        >
          Boo, we couldn't find that page.
        </h2>
        <p
          class="notFound__subtitle"
        >
          Not to worry, restring and move on! Nothing to see here...
        </p>
        <a
          aria-label="Navigate to home"
          class="notFound__logoContainer"
          href="/"
        >
          <app-logo-with-text>
            <svg
              class="svg"
              viewBox="0 0 2000 2000"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M302.39 1410.89v42.7h61.43v15.05h-61.43v49.53h-17.5v-122.5h86.45v15.22zM499.93 1518.17l-26.42-37.63c-3.27.24-5.84.35-7.7.35h-30.28v37.28H418v-122.5h47.78q23.79 0 37.45 11.37t13.65 31.33q0 14.18-7 24.15T490 1477l29 41.17zm-9.27-59.33q8.74-7.17 8.75-20.47t-8.75-20.37q-8.76-7.1-25.38-7.09h-29.75V1466h29.75q16.62 0 25.38-7.16zM658.74 1502.94v15.23h-88.9v-122.5h86.45v15.22h-68.95v37.63h61.43v14.87h-61.43v39.55zM739.79 1410.89h-42v-15.22h101.32v15.22h-42v107.28h-17.32zM861 1511.43a60.22 60.22 0 01-23.36-22.49 64.65 64.65 0 010-64A60.29 60.29 0 01861 1402.4a72.43 72.43 0 0166.68 0 59.84 59.84 0 0123.32 22.4 65.59 65.59 0 010 64.23 59.77 59.77 0 01-23.27 22.4 72.35 72.35 0 01-66.68 0zm57.67-13.56a45 45 0 0017-16.89 49.73 49.73 0 000-48.13 44.94 44.94 0 00-17-16.88 48.09 48.09 0 00-24.24-6.13A49 49 0 00870 1416a44.51 44.51 0 00-17.15 16.88 49.66 49.66 0 000 48.13 44.6 44.6 0 0017.15 16.86 49.1 49.1 0 0024.41 6.12 48.19 48.19 0 0024.28-6.12zM1116.21 1395.67v122.5h-14.35l-73.5-91.35v91.35h-17.5v-122.5h14.35l73.5 91.35v-91.35zM1256.49 1487.54h-65.1l-13.48 30.63h-18l55.47-122.5h17.33l55.65 122.5H1270zm-6.13-14l-26.42-60-26.43 60zM1343.06 1410.89h-42v-15.22h101.32v15.22h-42v107.28h-17.32zM1464.3 1511.43a60.17 60.17 0 01-23.37-22.49 64.71 64.71 0 010-64 60.24 60.24 0 0123.37-22.49 72.41 72.41 0 0166.67 0 59.8 59.8 0 0123.28 22.4 65.59 65.59 0 010 64.23 59.73 59.73 0 01-23.28 22.4 72.33 72.33 0 01-66.67 0zm57.66-13.56a45 45 0 0017-16.89 49.66 49.66 0 000-48.13 44.94 44.94 0 00-17-16.88 48.09 48.09 0 00-24.24-6.13 49 49 0 00-24.41 6.13 44.58 44.58 0 00-17.15 16.88 49.73 49.73 0 000 48.13 44.67 44.67 0 0017.15 16.89 49.13 49.13 0 0024.41 6.12 48.19 48.19 0 0024.28-6.12zM1696 1518.17l-26.42-37.63c-3.27.24-5.84.35-7.7.35h-30.28v37.28h-17.5v-122.5h47.78q23.79 0 37.45 11.37t13.65 31.33q0 14.18-7 24.15t-20 14.52l29 41.13zm-9.27-59.33q8.74-7.17 8.75-20.47t-8.75-20.39q-8.76-7.1-25.38-7.09h-29.75V1466h29.75q16.65 0 25.41-7.16zM1080.78 1060.28L1134 968a14.91 14.91 0 002-7.49V487.93a7.5 7.5 0 00-7.5-7.5h-54.18a7.5 7.5 0 00-7.5 7.5v568.61a7.5 7.5 0 0013.96 3.74zM1229.85 480.43h-54.19a7.5 7.5 0 00-7.5 7.5v392.72a7.5 7.5 0 0014 3.75l53.18-92.27a15 15 0 002-7.49V487.93a7.5 7.5 0 00-7.49-7.5zM925.71 480.43h-54.18a7.5 7.5 0 00-7.5 7.5v472.59a14.91 14.91 0 002 7.49l53.18 92.27a7.5 7.5 0 0014-3.74V487.93a7.5 7.5 0 00-7.5-7.5zM1363.08 480.43h-86a7.5 7.5 0 00-7.5 7.5v216.84a7.5 7.5 0 0014 3.74l105.53-183.09a30 30 0 00-26.03-44.99zM764.66 792.13l53.18 92.27a7.5 7.5 0 0014-3.75V487.93a7.5 7.5 0 00-7.5-7.5h-54.19a7.5 7.5 0 00-7.5 7.5v296.71a15 15 0 002.01 7.49zM1027.09 480.43h-54.18a7.5 7.5 0 00-7.5 7.5v648.48a15 15 0 002 7.49l6.6 11.44a30 30 0 0052 0l6.6-11.44a15 15 0 002-7.49V487.93a7.5 7.5 0 00-7.52-7.5zM730.46 487.93a7.5 7.5 0 00-7.5-7.5h-86a30 30 0 00-26 45l105.5 183.08a7.5 7.5 0 0014-3.74z"
              />
            </svg>
          </app-logo-with-text>
        </a>
      </div>
    </main>
  </app-not-found-index>
</app-not-found-index-spec>
`;
